mlrun/model.py

```diff
@@ -1014,7 +1014,7 @@ def __init__(
     ):
 
         self.name = name
-        self.path = str(path)
+        self.path = str(path) if path is not None else None
```

Comment:
isn't the opposite of not None = None?

Reply:
if path is not None
   self.path = str(path)
else
   self.path = None

Reply:
ok